src/chrome/content/rules/Scieneo.de.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+    Nonfunctional subdomains:
+	    - $
+
+    More rulesets for sites by Mirco Neubert:
+	    - Dynatech.de
+	    - Scieneo.de
```

Comment:
Yes, thanks. But AFAIK usually the ruleset names (without file extension) are used.